client/my-sites/stats/annual-highlights-section/yearNavigation.tsx

```diff
@@ -0,0 +1,59 @@
+import { Icon, arrowLeft, arrowRight } from '@wordpress/icons';
+import classNames from 'classnames';
+import { recordGoogleEvent } from 'calypso/state/analytics/actions';
+
+import './style.scss';
+
+type YearNavigationProps = {
+	onYearChange: ( next?: boolean ) => void;
+	disablePreviousArrow: boolean;
+	disableNextArrow: boolean;
+};
+
+const YearNavigation = ( {
+	onYearChange,
+	disablePreviousArrow,
+	disableNextArrow,
+}: YearNavigationProps ) => {
+	const handleClickArrow = ( next?: boolean ) => {
+		const arrow = next ? 'next' : 'previous';
+		recordGoogleEvent( 'Stats Period Navigation', `Clicked ${ arrow } year` );
+
+		onYearChange( next );
+	};
+
+	const onKeyDown = ( event: React.KeyboardEvent< HTMLButtonElement >, next?: boolean ) => {
+		if ( event.key === 'Enter' || event.key === 'Space' ) {
+			handleClickArrow( next );
+		}
+	};
+
+	return (
+		<div className={ classNames( 'stats-period-navigation', 'stats-year-navigation' ) }>
```

Comment:
Maybe we can polish the component `StatsPeriodNavigation` to reuse it in the future.

Reply:
I suspect Dognose is referring to the back/forward arrow pattern being present throughout multiple stats pages. I agree it would be nice to refactor the back/forward arrows into a reusable component.

Reply:
That would be handy. I'll add a task to our maintenance board if you don't mind.

Reply:
Sounds good to me!